src/Pos/Client/Txp/Balances.hs

```diff
@@ -68,17 +69,28 @@ runBalancesRedirect = coerce
 
 instance
     ( MonadDBRead m
+    , MonadRealDB m
     , WebWalletModeDB m
-    ,  MonadMask m
+    , MonadMask m
     , WithLogger m
     , MonadTxpMem ext m, t ~ IdentityT) =>
         MonadBalances (Ether.TaggedTrans BalancesRedirectTag t m)
   where
-    getOwnUtxos addr = do
+    getOwnUtxos addrs = do
+        let isRedeem (RedeemAddress _) = True
```

Comment:
It wouldn't make it much cleaner and actually the opposite - it would take more time for eyes to find `isRedeem`. Not worth restarting CI :)